packages/hops/mixins/analyzer/mixin.core.js

```diff
@@ -0,0 +1,32 @@
+const { Mixin } = require('hops-mixin');
+
+module.exports = class AnalyzerCoreMixin extends Mixin {
+  configureCommand(definition) {
+    if (definition.command === 'build') {
+      definition.builder.analyzeClientBundle = {
+        type: 'boolean',
+        default: false,
+        describe: 'Opens webpack bundle analyzer page after client side build',
```

Comment:
This description is still mentioning that the analyzer will be opened automatically - that should be changed.

Reply:
Good catch..!